prometheus/package.json

```diff
@@ -1,7 +1,6 @@
 {
-  "name": "@perses-dev/prometheus",
-  "private": true,
-  "version": "0.5.1",
+  "name": "@perses-dev/prometheus-plugin",
```

Comment:
I am wondering if we shouldn't keep `@perses-dev/prometheus` to show the changes and mark the previous package deprecated.
WDYT ?

Reply:
Yeah I thought about it. But then the packages won't have the plugin name on it. At the end they have the same content but in a breaking change fashion 😜 . Should use the `@perses-dev/{plugin_name}-plugin` format also for other plugins?

Reply:
well if we want to go this way, yeah I guess we need to have this naming convention to make the difference with the core package.

but that means next time for prometheus and tempo we will have to tag them with the version v0.51.0 insteand of v0.6.0 or equivalent